lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -255,4 +310,24 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
   defp assign_form(socket, %Ecto.Changeset{} = changeset) do
     assign(socket, form: to_form(changeset))
   end
+
+  defp assign_dockerfile(socket) do
+    form = socket.assigns.dockerfile_form
+
+    version = to_string(Application.spec(:livebook, :vsn))
+    version = if version =~ "dev", do: "edge", else: version
+    apps_path = if value = form[:apps_path].value, do: value, else: "/apps"
+
+    assign(socket, :dockerfile, """
+    FROM livebook/livebook:#{version}
+
+    ENV LIVEBOOK_APPS_PATH "#{apps_path}"
```

Comment:
I think this is not enough right? We need something like this:

```suggestion
    COPY /path/to/my/notebooks /data
    ENV LIVEBOOK_APPS_PATH "/data"
```

And then we need to tell them to replace /path/to/my/notebooks, right? But the COPY command will fail anyway if the directory is missing, so may no apps_path input at all?